src/denovo_discovery/local_assembly.cpp

```diff
@@ -109,28 +125,17 @@ void get_paths_between_util(const std::string &start_kmer, const std::string &en
     auto &child_nodes = tree[start_kmer];
```

Comment:
perhaps move this alias closer to where it's used?: after the if statement below